lib/guest_installation_and_configuration_base.pm

```diff
@@ -394,10 +394,10 @@ sub prepare_non_transactional_environment {
     if (!is_transactional) {
         virt_autotest::utils::setup_rsyslog_host($common_log_folder);
         my $_packages_to_check = 'wget curl screen dnsmasq xmlstarlet yast2-schema python3 nmap';
-        zypper_call("install -y $_packages_to_check");
+        zypper_call("install -y $_packages_to_check", exitcode => [0, 106]);
         my $_patterns_to_check = 'kvm_server kvm_tools';
         $_patterns_to_check = 'xen_server xen_tools' if ($self->{host_virt_type} eq 'xen');
-        zypper_call("install -y -t pattern $_patterns_to_check");
+        zypper_call("install -y -t pattern $_patterns_to_check", exitcode => [0, 106]);
```

Comment:
Would you please share what else repositories failed to be refreshed in addition to virtupdate? In this PR, the allowed zypper errors mainly come from repository side. If the repositories do not provide packages that we'd like to test, eg kvm,xen,libvirt, allowing such errors can be fine, eg above line 397 change. But Virtupdate repo that we set is the official product update repo which provides updated versions of kvm,xen,libvirt. I am wondering if such repo has problem, shall we stop the test? Lower versions of packages than update repo is not our test target.

Reply:
autobuild guys fixed the issue. Removed "exitcode". Done. Verification updated.